common/plugins/eu.esdihumboldt.hale.common.instance.graph/src/eu/esdihumboldt/hale/common/instance/graph/reference/ReferenceGraph.java

```diff
@@ -294,23 +300,36 @@ public void remove() {
 
 	private boolean partitioned = false;
 
+	private final T handleFirst;
+
 	/**
 	 * Create a new reference graph from the given instance collection.
 	 * 
 	 * @param inspector the instance inspector to use
 	 * @param instances the
+	 * @param handleFirst first to be handle
```

Comment:
Maybe more detailed?
It should be clear that what is passed in here is an ID, and that it is optional and null may be passed in.